src/Committee/Election/CandidacyManager.php

```diff
@@ -112,4 +121,34 @@ public function updateInvitation(
             new CandidacyInvitationEvent($candidacy, $invitation, $previouslyInvitedMembership)
         );
     }
+
+    public function acceptInvitation(CommitteeCandidacyInvitation $invitation, CommitteeCandidacy $acceptedBy): void
```

Comment:
[question] Isn't `$acceptedBy` already attached to `$invitation`, if yes, no need to require the second argument

Reply:
no, $invitation does not have this object